setup/www/resources/config/libuv.org

```diff
@@ -1,14 +1,26 @@
 server {
     listen *:80;
-    listen [::]:80;
```

Comment:
Correct me if I'm wrong, but this should be safe to add back in because ultimately the server is selected by the server_name token, right?

Aside: it can probably be generalized to e.g.
```
server_name dist.libuv.org, dist.nodejs.org, ...;
return 301 https://$server_name$request_uri;
```

Reply:
FYI yes it's been added back in in #1402